card/spec/lib/card/query/card_query/special_attributes_spec.rb

```diff
@@ -0,0 +1,76 @@
+require_relative "../query_spec_helper"
+RSpec.describe Card::Query::CardQuery::SpecialAttributes do
+  include QuerySpecHelper
+
+  # TODO: add specs for: #name_match, #complete, #junction_complete
+
+  describe "match" do
+    it "reachs content and name via shortcut" do
+      @query = { match: "two" }
+      is_expected.to eq(cards_matching_two)
```

Comment:
The style guide (which is disabled for theses files) dislikes instances variables and he has a point.
It obfuscates what the test is doing.

Reply:
I agree. This PR didn't introduce that, though. Can we go ahead and merge this?  I want to tackle the actual performance problems before getting into more spec clean up.

Reply:
ok, I took care of it.  should be ready now.